cmd/namespace/use.go

```diff
@@ -99,7 +99,9 @@ func (nc *NamespaceCommand) getNamespaceFromSelector(ctx context.Context) (strin
 	if err != nil {
 		return "", err
 	}
-	ns, _, err := utils.AskForOptionsOkteto(ctx, namespaces, "Select the namespace you want to use:", "Namespace")
+	initialPosition := getInitialPosition(namespaces)
+	selector := utils.NewOktetoSelector("Select the namespace you want to use:", "Namespace")
+	ns, err := selector.AskForOptionsOkteto(namespaces, initialPosition)
```

Comment:
WDYT about creating a functioni for this? I've seen that we have this pattern a few times. I think it's a sign that most of the times we are interested in the `ns` outcome of this sequence, and unifying it in one function call would simplify the api from a consumer standpoint

Reply:
i decoupled the selector creation and the question asked to be able to fake the selector behaviour. In any case we use `AskForOptionsOkteto` we want to have as a result a string with the selection, this is a repeteated pattern in every request for user information we do.

if you refer to the initialPosition func, this is required at context and namespace selection so the selector prompt starts with the arrow at the current one.